apps/site/src/app/admin/applicants/(summary)/ApplicationsBySchoolChart.tsx

```diff
@@ -0,0 +1,84 @@
+import BarChart from "@cloudscape-design/components/bar-chart";
+import Box from "@cloudscape-design/components/box";
+
+import useApplicationsSummary from "./useApplicationsSummary";
+
+const TIME_SPEC = "T00:00:00-08:00";
+const START_DAY = new Date("2024-12-11" + TIME_SPEC);
+const END_DAY = new Date("2025-01-11" + TIME_SPEC);
+
+// In reverse of desired order
+const KNOWN_SCHOOLS = [
+	"UC San Diego",
+	"UCLA",
+	"UC Riverside",
+	"Cal State Fullerton",
+	"Cal State Long Beach",
+	"Orange Coast College",
+	"UC Irvine",
+];
```

Comment:
Just curious, I'm guessing IH (formerly HackUCI) historically have received the most apps from these schools?

Reply:
In a way, based on a subset of the [predefined options in `SchoolInformation`](https://github.com/HackAtUCI/irvinehacks-site/blob/79a9a046d2508d504fd56d71d983b711062d8a1f/apps/site/src/lib/components/forms/shared/SchoolInformation.tsx#L14), perhaps most frequent but not exactly, and partly to fit with default colors set by Cloudscape. Feel free to modify as desired.

I previously didn't have this, but this year, the first person who applied was actually not from UCI, and I wanted to make UCI the light blue color. (Was partly mitigated last year since a logistics director submitted the first test app in production).